releaseNotes/MicrosoftExtensionsConfigurationAzureAppConfiguration.md

```diff
@@ -1,14 +1,13 @@
 ## Microsoft Extensions Azure App Configuration
 
 ### v1.0.0-preview-008920001-990
-
+#### Released on: June 11<sup>th</sup>, 2019
```

Comment:
Can we just add date after the version to be consistent with other release notes (eg. https://github.com/Azure/AppConfiguration/blob/master/releaseNotes/AzureDevOpsPipelineExtension.md)?